docs/introduction/Car-registry-tutorial.rst

```diff
@@ -232,22 +237,21 @@ A SellOrderProposition is implemented as
 ::
  public final class SellOrderProposition implements ProofOfKnowledgeProposition<PrivateKey25519>
 
-A point to note is that the proposition contains two public keys, thus that proposition could be opened by two different keys.
+A point to note is that the proposition contains two public keys, thus that proposition could be opened by two different private keys.
 
 Implementation of SellOrderPropositionSerializer
 ************************************************
 A SellOrderPropositionSerializer is implemented as 
 ::
  public final class SellOrderPropositionSerializer implements PropositionSerializer<SellOrderProposition>
 
-
 Implementation of SellOrderSpendingProof  
 ****************************************
 A SellOrderSpendingProof is implemented as  
 ::
  extends AbstractSignature25519<PrivateKey25519, SellOrderProposition>
 
-Implementation Comments: Information about proof type is defined by the result of method boolean isSeller(). For example an implementation of method isValid uses that flag:
+Implementation Comments: Information about proof type is defined by the result of method boolean isSeller(). For example an implementation of method isValid uses the flag:
```

Comment:
Implementation Comments: Information about the proof type is defined by the result of a method boolean isSeller(). For example an implementation of the method isValid uses the flag: